### DISCUSSION TITLE
[RESOLVED] A minor development pause (my entire PC died, lol)

### DISCUSSION THREAD

**Discussion by SirSquidly** [SirSquidly/Oceanic-Expanse#58] [Announcements]
Recently, I have been attempting to work more consistently on Oceanic Expanse by pushing daily commits to the GitHub. This was so I can try to keep a consistent stream of progress, where something is always getting done, fixed, added, or tweaked.

I am forced to break my 17 day streak of commits, as my CPU Cooler decided to die on me. 

I ordered a new part, which should be arriving tomorrow, wherein I will get right back to work.

I hope you all have a wonderful day.

EDIT: This has been resolved. Title has been changed from 'A minor development pause (my entire PC died, lol)' to reflect this.

**Comment by vico93** [comment]
Hope you can get through this soon!

**Reply by SirSquidly** [reply]
Thank you for the kind words.

I am awaiting my part currently.
